### PR TITLE
[v9] Record when a session recording is accessed

### PR DESCRIPTION
**Pull request by ryanclark**
Backport #14302 to branch/v9

### CODE REVIEW

**Review by strideynet — APPROVED** [review]

File: lib/auth/auth_with_roles_test.go [hunk 1]
```diff
@@ -35,6 +35,11 @@ import (
 	"github.com/gravitational/teleport/api/utils/sshutils"
 	"github.com/gravitational/teleport/lib/auth/native"
 	libdefaults "github.com/gravitational/teleport/lib/defaults"
+<<<<<<< HEAD
+=======
+	"github.com/gravitational/teleport/lib/events"
+	"github.com/gravitational/teleport/lib/fixtures"
+>>>>>>> 1d269b09b (Record when a session recording is accessed)
```
Comment (espadolini) [inline lib/auth/auth_with_roles_test.go:42]: There's a leftover merge conflict here

**Review by espadolini — APPROVED** [review]

File: api/types/events/events.proto [hunk 2]
```diff
@@ -1996,6 +1996,7 @@ message OneOf {
         events.MySQLDebug MySQLDebug = 86;
         events.MySQLRefresh MySQLRefresh = 87;
         events.SFTP SFTP = 88;
+        events.SessionRecordingAccess SessionRecordingAccess = 89;
```
Comment (espadolini) [inline api/types/events/events.proto:1999]: The message number should match the one in master, it's fine to skip them here

**Review by espadolini — CHANGES_REQUESTED** [review]

File: lib/auth/auth_with_roles_test.go (hunk 1 above)
Comment (ryanclark) [reply lib/auth/auth_with_roles_test.go:42]: Oops..

File: api/types/events/events.proto (hunk 2 above)
Comment (ryanclark) [reply api/types/events/events.proto:1999]: Ah okay, I wasn't sure

**Review by espadolini — APPROVED** [review]

**Review by zmb3 — APPROVED** [review]

**Pull request merged**